docs/release-notes/highlights-6.3.0.asciidoc

```diff
@@ -1,9 +1,124 @@
 [[release-highlights-6.3.0]]
-== 6.3.0 release highlights
+== 6.3.0 Release Highlights
 ++++
 <titleabbrev>6.3.0</titleabbrev>
 ++++
 
 coming[6.3.0]
 
-See also <<breaking-changes>> and <<release-notes-6.3.0>>.
\ No newline at end of file
+Each release of {kib} brings new features and product improvements. Here are the 
+highlights of the features that were added in 6.3 and how the user experience 
+improved.
+
+Refer to the <<release-notes-6.3.0, {kib} 6.3 Release Notes>> for a list bug 
+fixes and other changes.
+
+[float]
+=== License management and {xpack} code
+
+Starting with version 6.3, all of the free {xpack} features, such as monitoring 
+and tile maps, ship with the default distribution of {kib}.  When you install 
```

Comment:
OSS actually has tile maps, it just doesn't have as many out of the box zoom levels. There are other things you can call out though. Basically, anything that is under the `Basic` column in the subscription page [here](https://www.elastic.co/subscriptions)

I'd maybe focus on things like CSV export, the dedicated API UI and additional dev tools (grok debugger, search profiler)